src/Bridge/Doctrine/EventListener/WriteListener.php

```diff
@@ -58,7 +58,9 @@ public function onKernelView(GetResponseForControllerResultEvent $event)
 
         switch ($request->getMethod()) {
             case 'POST':
-                $objectManager->persist($controllerResult);
+                $event->setControllerResult(
```

Comment:
CS: We usually don't split into multiple lines like this... Just keep it in one line.

Reply:
I won’t argue on CS, but in this case, given the low visual impact of `??`, I fear that relegating the `??` at the end of the line somehow hide what’s going on. 

For instance, to me, this is far less self-explanatory than the splitted version.
 
```
    case 'POST':
                $event->setControllerResult($this->dataPersister->persist($controllerResult) ?? $controllerResult));
                break;
```

As I said, this will be my only point of view on CS, I wont argue furthermore if you want to stick to the one line version ;)

Reply:
nevermind, adding the deprecation notice forces me to assign persist result to a temp var anyway, so the one line will be of fine size :+1: